src/Microsoft.OpenApi.OData.Reader/Operation/ComplexPropertyDeleteOperationHandler.cs

```diff
@@ -1,90 +0,0 @@
-// ------------------------------------------------------------
```

Comment:
why delete?

Reply:
> why delete?

Same as above, the reason has been mentioned in the issue: https://github.com/microsoft/OpenAPI.NET.OData/issues/176